lib/lambda_ethereum_consensus/p2p/incoming_requests/handler.ex

```diff
@@ -85,9 +85,9 @@ defmodule LambdaEthereumConsensus.P2P.IncomingRequests.Handler do
 
   defp handle_req("metadata/2/ssz_snappy", message_id, _message) do
     # Values are hardcoded
-    with {:ok, payload} <-
-           <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0>>
-           |> Snappy.compress() do
+    with metadata <- P2P.Metadata.get_metadata(),
```

Comment:
We can leave the metadata assignment outside of the with, as we don't need to match its contents. We can do either:

```elixir
metadata = P2P.Metadata.get_metadata()
with {:ok, payload} <-  Ssz.to_ssz(metadata),
#...
```

or 

```elixir
with {:ok, payload} <- P2P.Metadata.get_metadata() |> Ssz.to_ssz(),
#...